### ISSUE TITLE
Have the shaking hearts be % based

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#2898]
The hearts shake at 4 HP or lower, which is 20% of vanilla's max HP. I would like this 20% equivalent to stay the same no matter how much your HP increases. If I'm in the middle of an intense fight, I won't have time to look at the exact number and gauge its meaning, I can only look at the heart, and if I get down to 4/80 HP and it starts shaking, I may be dead before I can comphrehend the heart shaking.

To recap, make it so the heart starts shaking at **20%** HP or lower, not just 4 raw HP or lower
